<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="ragini" failures="1" tests="6" name="Default test" time="77.661" errors="0" timestamp="2024-11-27T18:30:12 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="9.621"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="9.116"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="9.816"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="5.093">
    <failure type="java.lang.AssertionError" message="expected [] but found [Your username is invalid!]">
      <![CDATA[java.lang.AssertionError: expected [] but found [Your username is invalid!]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at StepDefinitions.LoginStepsDefinition.user_should_be_shown_invalid_credential_message(LoginStepsDefinition.java:108)
at âœ½.User should be shown invalid credential message(file:///D:/BDD_Workspace/BDDFramework/src/test/resources/Features/login.feature:22)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.89"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="35.183"/>
</testsuite> <!-- Default test -->
